orttraining/orttraining/core/graph/optimizer/adam_optimizer_builder.cc

```diff
@@ -156,7 +160,7 @@ Status AdamOptimizerBuilder::Build(
                                       BuildAttributeProto(opt_configs[i]),
                                       OptimizerNodeName(weight_name))});
     }
-
+    
```

Comment:
can you run a formatter

Reply:
I ran the VS code formatter, somehow it didn't fix this but changed format somewhere else.
So I removed this whitespace manually